src/mdns/mdns_avahi.cpp

```diff
@@ -33,6 +33,7 @@
 
 #include "mdns/mdns_avahi.hpp"
 
+#include <algorithm>
```

Comment:
nit - separate into own block:
```suggestion
#include <algorithm>

```